### PR TITLE
Add rubygem-foreman_netbox package

### PR DESCRIPTION
**Pull request by dgoetz**

### CODE REVIEW

**Comment by dgoetz** [comment]
And it is green now!

**Review by evgeni — CHANGES_REQUESTED** [review]
The min version needs adjustment (and the comment above it can go away then).

Otherwise lgtm

File: packages/plugins/rubygem-foreman_netbox/rubygem-foreman_netbox.spec [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#   2. Delete these lines
+#
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name foreman_netbox
+%global plugin_name netbox
+%global foreman_min_version FIXME
```
[the remark this answers is not in the document]
Comment (dgoetz) [reply packages/plugins/rubygem-foreman_netbox/rubygem-foreman_netbox.spec]: I forgot it again! :-( Fixed now!

**Review by evgeni — COMMENTED** [review]
ACK

I don't have access to koji right now, so can't add the pkg and merge, but someone else might get to it later ;)

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
Really, nobody cared to finish this off? :disappointed: 

```console
% koji add-pkg --owner kojiadm foreman-plugins-nightly-el8 rubygem-foreman_netbox   
Adding 1 packages to tag foreman-plugins-nightly-el8
% koji add-pkg --owner kojiadm foreman-plugins-nightly-rhel7 tfm-rubygem-foreman_netbox
Adding 1 packages to tag foreman-plugins-nightly-rhel7
```

**Pull request merged**